Multisig/Features/Connect to Wallet/Data/Registry Service/WCGetDappsRequest.swift

```diff
@@ -7,7 +7,7 @@ import Foundation
 
 struct WCGetDappsRequest: JSONRequest {
     var httpMethod: String { "GET" }
-    var urlPath: String { "/data/dapps.json" }
+    var urlPath: String { "/safe-ios-mvp-hosting/data/dapps.json" }
```

Comment:
I would leave this as it was '/data/dapps.json' and change WALLETCONNECT_REGISTRY_URL to include /safe-ios-mvp-hosting/ if possible

Reply:
unfortunately this didn't work, I had to use the full path. Tech limitation of the current json http client